exercises/luhn/src/example.erl

```diff
@@ -34,14 +48,26 @@ checksum([H | ReversedNumber], even, Total) when H >= $5 ->
 
 
 valid(Number) ->
-  checksum(Number) rem 10 == 0.
-
-
-
-create(Number) ->
-  lists:flatten([Number, ($: - (checksum(Number ++ [$0]) rem 10))]).
+  case is_numeric(Number) of
+    true ->
+      case is_min_length(Number) of
+      	true ->
+	      case lists:filter(
+	        fun(C) ->
+	            ($0 =< C) andalso (C =< $9)
+	        end,
+	        Number
+	      ) of
+	        Number2 = [_|_] ->
+	          checksum(Number2) rem 10 == 0;
+	        _ -> false
+	      end;
+        _ -> false
+      end;
+    _ -> false
+ end.
 
 
 
 test_version() ->
-    1.
+    1.1.
```

Comment:
Please no floating points for the version.

I generally stick to integers here and increment by `1` each time. That would make this `2`.